packages/ods/src/components/badge/src/constants/badge-color.ts

```diff
@@ -1,7 +1,13 @@
 enum ODS_BADGE_COLOR {
+  neutral = 'neutral',
   information = 'information',
   success = 'success',
   warning = 'warning',
+  critical = 'critical',
+  alpha = 'alpha',
+  beta = 'beta',
+  new = 'new',
+  promotion = 'promotion',
```

Comment:
please try to keep alphabetical order everywhere (scss mixin also), it makes things easier to read later on